scene/gui/line_edit.cpp

```diff
@@ -1318,6 +1318,23 @@ void LineEdit::select_all() {
 void LineEdit::set_editable(bool p_editable) {
 
 	editable = p_editable;
+
+	// Reorganize context menu.
```

Comment:
Maybe add a
```
if (editable == p_editable)
	return;
```
to prevent clearing the menu needlessly. Same for TextEdit.

Reply:
Done.